movie-browser/src/App.js

```diff
@@ -1,31 +1,61 @@
 /* eslint-disable no-unused-vars */
 import React from 'react'
-import CardContainer from './CardContainer'
+import CardContainer from './view/CardContainer'
+import FeaturedMovie from './view/FeaturedMovie'
 import ApiReader from './model/ApiReader';
 
 // *** constant(s)
 const url = 'https://ghibliapi.herokuapp.com/films'
+const btnTxtDef = "View all"
+
+// TODO: fix sorting by year. have no idea why it works for score but not for year
```

Comment:
shouldn't it be `release_date` and not date ?

Reply:
Thanks for that. I have no idea what I was thinking.
Compare to Java, Javascript throws too less error.